docs/contributing/widgets/testing.md

```diff
@@ -39,15 +98,27 @@ describe('Node-RED Dashboard 2.0 - Buttons', () => {
 
     it('can be clicked and outputs the correct payload & topic are emitted', () => {
         // Emitting strings
-        cy.get('button').contains('Button 1 (str)').click()
+        cy.clickAndWait(cy.get('button').contains('Button 1 (str)'))
         // checkOutput calls our helper endpoints to checks the values against the stored msg
         cy.checkOutput('msg.payload', 'button 1 clicked')
         cy.checkOutput('msg.topic', 'button-str-topic')
 
         // Emitting JSON
-        cy.get('button').contains('Button 1 (json)').click()
+        cy.clickAndWait(cy.get('button').contains('Button 1 (json)'))
         cy.checkOutput('msg.payload.hello', 'world')
         cy.checkOutput('msg.topic', 'button-json-topic')
     })
 })
-```
\ No newline at end of file
+```
+
+### Rest Context
+
+`cy.resetContext()`
+
+The above functions to assist with setting and checking output rely on Node-RED's "context" stores. If at any point you want to be sure they're reset and cleared, this function will clear the context stores, before proceeding to any further steps in your tests.
```

Comment:
Not sure what this means Joe.

I get that the function `resetContext` clears Node-REDs context stores but the wording around "to assist with settings output rely" is every verbose and making me lose focus or dont have context (sic) to understand it.